packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/useDatabases.ts

```diff
@@ -36,12 +36,20 @@ export function useDatabases() {
       // `port` is not passed as well, we'll let the tsh daemon pick a random one.
       targetUri: db.uri,
       targetName: db.name,
-      targetUser: dbUser,
+      targetUser: getTargetUser(db.protocol, dbUser),
     });
     documentsService.add(doc);
     documentsService.open(doc.uri);
   }
 
+  function getTargetUser(protocol: string, providedDbUser: string): string {
+    if (protocol === 'redis') {
+      return providedDbUser || 'default';
```

Comment:
Could you mention in a comment that we're replicating [tsh logic](https://github.com/gravitational/teleport/blob/796e37bdbc1cb6e0a93b07115ffefa0e6922c529/tool/tsh/db.go#L240-L244) here but unlike tsh, in Connect each gateway document assumes that the username is present?

Reply:
Done